staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_tls_test.go

```diff
@@ -172,6 +178,7 @@ func TestCreateSecretTLS(t *testing.T) {
 				Name:       test.tlsSecretName,
 				Key:        test.tlsKey,
 				Cert:       test.tlsCert,
+				Ca:         test.tlsCa,
```

Comment:
Once we add some CA validation, I guess it would be good to use some invalid CA and test that validation as well

Reply:
I added some validation.  Please check whether it is what you had in mind.